input/input.css

```diff
@@ -0,0 +1,122 @@
+@import url(/CSS/nav.css);
+
+:root {
+
+    --incorrect-password: #dc2626;
```

Comment:
Be more generic with variable naming so that it can be re-used

Reply:
Makes sense, thanks Nikhil